src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -43,16 +78,21 @@ public static Query buildSearchConceptsQuery(
             true);
 
     // domain clause filters for the given domain id based on field domain_id
+    // c.domain_id = 'domain'
     var domainClause =
         createDomainClause(conceptTablePointer, conceptTableVariable, domainOption.getName());
 
+    // c.concept_name, c.concept_id, COUNT(DISTINCT co.person_id) AS count
     List<SelectExpression> select = List.of(nameField, idField, countField);
 
-    List<TableVariable> tables = List.of(conceptTableVariable, domainOccurenceTableVariable);
+    List<TableVariable> tables =
+        List.of(conceptTableVariable, conceptAncestorTableVariable, domainOccurenceTableVariable);
```

Comment:
This is a place where pulling the `TableVariable` part of the variable names would make it read much more cleanly. For example:
`List<TableVariable> tables = List.of(concept, conceptAncestor, domainOccurrence);`